css/styles.css

```diff
@@ -0,0 +1,602 @@
+  gap: 20px;
+  margin-top: 2rem;
+}
+
+.workshop-1 {
+  background: linear-gradient(rgba(236, 82, 66, 0.7), rgba(39, 42, 49, 0.6)), url(../images/workshop-1.png) no-repeat center;
+  background-size: cover;
+  height: 10rem;
+  width: 100%;
+  text-align: center;
+  color: #fff;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.workshop-1:hover {
+  background: linear-gradient(rgba(255, 255, 255, 0.1), rgba(255, 255, 255, 0.1)), url(../images/workshop-1.png) no-repeat center;
+}
+
+.workshop-2 {
+  background: linear-gradient(rgba(236, 82, 66, 0.7), rgba(39, 42, 49, 0.6)), url(../images/workshop-2.png) no-repeat center;
+  background-size: cover;
+  height: 10rem;
+  width: 100%;
+  text-align: center;
+  color: #fff;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.workshop-2:hover {
+  background: linear-gradient(rgba(255, 255, 255, 0.1), rgba(255, 255, 255, 0.1)), url(../images/workshop-2.png) no-repeat center;
+}
+
+.footer {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: #272a31;
+  padding: 10px;
+}
+
+.footer img {
+  width: 45%;
+  height: auto;
+}
+
+@media (min-width: 48rem) {
+  body {
+    background-image: url(../images/bg-3.jpg);
+  }
+
+  .nav-bar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 20px;
+    position: fixed;
+    top: 0;
+    background-color: #fff;
+    width: 100%;
+    height: 70px;
+    z-index: 1;
+  }
+
+  .menu {
+    display: flex;
+    background: none;
+    gap: 20px;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    opacity: 0.8;
+    width: 100%;
+    height: 80px;
+    text-align: center;
+    transition: 0.3s;
+    padding-top: 10px;
+  }
+
+  .logo img {
+    display: block;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  section.hero {
+    padding: 40px 100px 20px 100px;
+  }
+
+  .hero-text {
+    font-size: 1.7rem;
+  }
+
+  .hero-header {
+    font-size: 3.5rem;
+    line-height: 1.2;
+  }
+
+  .description {
+    padding-right: 200px;
+  }
+
+  .header-h2 {
+    margin-top: 10px;
+    opacity: 0.8;
+  }
+
+  .main-program {
+    padding: 3rem 15rem;
+  }
+
+  .program-heading::after {
+    width: 8%;
+  }
+
+  .card-container {
+    flex-direction: row;
+    gap: 1.5rem;
+  }
+
+  .card {
+    flex-direction: column;
+    gap: 0.5rem;
+    padding: 1.3rem;
+  }
+
+  .card-text {
+    text-align: center;
+  }
+
+  .team {
+    padding: 2rem 5rem;
+  }
+
+  .team-container {
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .partners {
+    padding: 50px 0 100px 0;
+  }
+
+  .partners-header::after {
+    width: 6%;
+  }
+
+  .partner-list {
+    margin-top: 40px;
+  }
+
+  .partner-img {
+    width: 100px;
+    height: 35px;
+    margin-top: 100px;
+  }
+
+  .hero-about {
+    justify-content: center;
+  }
+
+  .detail {
+    width: 60%;
+    text-align: left;
+  }
+
+  .workshop {
+    flex-direction: row;
+  }
```

Comment:
- Hey, to control the width of each workshop, let's add a widht of about 65% to almost be close to the design template, thereby aligning the card in the center below:

![Screenshot from 2022-10-19 09-32-31](https://user-images.githubusercontent.com/59397159/196640518-6435fcb5-62e7-435f-9a6b-8b9fd448fab8.png)